babelsberg/constraintinterpreter.js

```diff
@@ -1099,8 +1154,8 @@ Object.subclass('ConstrainedVariable', {
                     }
             }
 
-            if (this._constraints.length > 0 && !hasEnabledConstraint)
-                return;
+            // if (this._constraints.length > 0 && !hasEnabledConstraint)
+            //     return;
```

Comment:
I don't exactly remember, but wasn't this some magic we added to get tests green and stuff? If the issue is solved now these lines can go entirely